less/mosaico/components/sharing.less

```diff
@@ -0,0 +1,17 @@
+.mosaico-article__some-tags-group {
+  display: flex;
+  list-style: none;
+  padding: 0;
+
+  > li {
```

Comment:
Maybe one thing you could do is moving the content of this selector into one named `.mosaico-article__tag` and apply the class names to every list element, but both versions are fine I think